.github/workflows/golangci-lint.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
       - name: Install Go
-        uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
```

Comment:
do we need this file with Dagger?

Reply:
no really but this file is being automatically synced by the prombot and @kakkoyun commented above that we might not want to modify it